lessons/Algebra2/linear2-building-models/langs/en-us/pages/model-college-v-income-1-scaffolded.adoc

```diff
@@ -3,47 +3,50 @@
 <style>
 /* Shrink fill in the blanks */
 .studentAnswerMedium { min-width: 8em !important; }
+.fitbruby {padding-top: 1rem !important;}
 </style>
 ++++
 
 [.linkInstructions]
 Open the @starter-file{alg2-states-intro}.
```

Comment:
I don't think it makes sense to have two separate files for this sequence. Let's discuss if you feel strongly about it.

Reply:
predefine a model for `mi-ca` in the starter file instead of giving way `al-ak`